django/contrib/sites/checks.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from django.core.checks import Error
+
+
+def check_site_id_type(**kwargs):
+    errors = []
+    if hasattr(settings, 'SITE_ID'):
+        if not isinstance(settings.SITE_ID, int):
+            errors.append(
```

Comment:
There is no need to `append()` because we have a single error:
```suggestion
            return [
```